vite.config.ts

```diff
@@ -38,6 +38,8 @@ export default defineConfig(({ mode }) => {
         fileName: (format) => `lib.${format}.min.js`
       } : undefined,
       terserOptions: {
+        keep_classnames: true,
```

Comment:
Vue component naming warning: Search$1 warning in console, saying it wasn't a valid html5 component. This changes it back to Search